### ISSUE TITLE
API handler doesn't set a statusCode for incorrect function names

### ISSUE THREAD

**Issue by raybellis** [ether/etherpad-lite#3546]
The REST API handler returns a 404 for an incorrect API number, and 401 for an incorrect API key, but doesn't change the status code if the specified function does not exist in the API.

**Comment by JohnMcLear** [comment]
@raybellis hrm, what should it be?

**Comment by raybellis** [comment]
I'd go for 404, since the function name also forms part of the URL path.

**Comment by JohnMcLear** [comment]
Ah it returns 200 or so at the moment?  Gimme example URI if you can, something easily fixable..

**Comment by raybellis** [comment]
I don't have a dev environment set up at the moment, but I think a one line change replacing the comment at https://github.com/ether/etherpad-lite/blob/develop/src/node/handler/APIHandler.js#L163 would suffice.

(This is just something I noticed during my async refactoring, but it was out of scope of those changes hence just flagging it as an issue).

**Comment by muxator** [comment]
Closed with a0579c90dbcc2677ef690d3149c1a28e3e39a73b because it was not possible to rebase the PR.

For documentation's sake, this would be a way for testing:

Before:
```bash
curl --verbose "http://localhost:9001/api/1/notExisting?apikey=ABCDEF"

< HTTP/1.1 200 OK
< X-Powered-By: Express
[...]
{"code":3,"message":"no such function","data":null}
```

After:
```bash
curl --verbose "http://localhost:9001/api/1/notExisting?apikey=ABCDEF"

< HTTP/1.1 404 OK
< X-Powered-By: Express
[...]
{"code":3,"message":"no such function","data":null}
```

Thanks @JohnMcLear for bringing this up and fixing it.
